### DISCUSSION TITLE
Mesh Boundary Error

### DISCUSSION THREAD

**Discussion by mikemcm22** [firemodels/fds#13755] [General]
![image](https://github.com/user-attachments/assets/4b0c016c-06ff-4856-962c-d7f977caccef)

I have always understood this to be a obstruction/mesh error. However, I have gone through and deleted an thin obstructions and checked my geometry in pyrosim. My meshes are aligned. What else can cause this error? A similar version of the model worked so I am not sure what could be causing this. I have no defined zones. Thanks.

**Comment by mcgratta** [comment]
Post the input file and I will take a look. This error occurs when FDS cannot figure out which zones are connected and how.

**Comment by mikemcm22** [comment]
I_DF_2A.fds
Generated by PyroSim 2023.3.1312
Nov 14, 2024, 12:47:17 PM

&HEAD CHID='I_DF_2A'/
&TIME T_END=1200.0/
&DUMP DT_RESTART=300.0, DT_SL3D=0.25/
&MISC VISIBILITY_FACTOR=0.8/

&MESH ID='FIRE MESH BOTTOM', IJK=60,66,66, XB=0.95,3.95,14.45,17.75,0.4,3.7/
&MESH ID='FIRE MESH BOTTOM 2', IJK=60,66,66, XB=3.95,6.95,14.45,17.75,0.4,3.7/
&MESH ID='FIRE MESH 1', IJK=99,33,63, XB=6.95,16.85,14.45,17.75,0.4,6.7/
&MESH ID='FIRE MESH 2', IJK=120,66,30, XB=0.95,6.95,14.45,17.75,3.7,5.2/
&MESH ID='FIRE MESH 3', IJK=120,66,30, XB=0.95,6.95,14.45,17.75,5.2,6.7/
&MESH ID='FIRE MESH COMBINED', IJK=159,33,42, XB=0.95,16.85,14.45,17.75,6.7,10.9/
&MESH ID='OUTER MESH 1', IJK=37,105,35, XB=16.85,27.95,0.05,31.55,0.4,10.9/
&MESH ID='OUTER MESH 2', IJK=53,48,35, XB=0.95,16.85,0.05,14.45,0.4,10.9/
&MESH ID='OUTER MESH 3', IJK=53,46,35, XB=0.95,16.85,17.75,31.55,0.4,10.9/

&SPEC ID='REAC_FUEL', FORMULA='C1.0H1.7O0.3N0.08'/

&REAC ID='Mattress',
      FUEL='REAC_FUEL',
      CO_YIELD=0.042,
      SOOT_YIELD=0.198,
      HEAT_OF_COMBUSTION=1.64E+4,
      RADIATIVE_FRACTION=0.537/

&PROP ID='Default',
      QUANTITY='LINK TEMPERATURE',
      ACTIVATION_TEMPERATURE=66.0,
      RTI=130.0/
&PROP ID='Cleary Photoelectric P1',
      QUANTITY='CHAMBER OBSCURATION',
      ALPHA_E=1.8,
      BETA_E=-1.0,
      ALPHA_C=1.0,
      BETA_C=-0.8/
&DEVC ID='Time', QUANTITY='TIME', XYZ=1.787201,0.35,1.245109, SETPOINT=0.0/
&DEVC ID='HD01', PROP_ID='Default', XYZ=7.35,28.25,9.7/

****Skipped past devices ******

&SURF ID='Global',
      COLOR='WHITE'/
&SURF ID='Fire',
      COLOR='RED',
      TEXTURE_MAP='psm_fire.jpg',
      HRRPUA=100.0,
      TAU_Q=-65.0,
      TMP_FRONT=300.0/

&OBST ID='FIRE MATRESS', XB=2.725,4.725,15.225,16.225,1.2,1.7, SURF_IDS='Fire','INERT','INERT'/ 
&OBST ID='Added Walls', XB=1.85,5.529541,24.35,24.75,1.3,6.7, COLOR='WHITE'/ 
&OBST ID='Added Walls', XB=1.55,4.366336,21.95,22.35,1.3,6.7, COLOR='WHITE'/ 
&OBST ID='AcDb3dSolid - 8A', XB=1.85,4.45,14.45,17.75,9.5,9.6, COLOR='INVISIBLE', SURF_ID='INERT'/ 
&OBST ID='AcDb3dSolid - 8A', XB=4.45,6.55,14.45,17.75,9.5,9.7, COLOR='INVISIBLE', SURF_ID='INERT'/ 
&OBST ID='AcDb3dSolid - 8A', XB=6.55,9.15,14.45,17.75,9.6,9.7, COLOR='INVISIBLE', SURF_ID='INERT'/ 
&OBST ID='AcDb3dSolid - 8A', XB=9.15,11.25,14.45,17.75,9.6,9.8, COLOR='INVISIBLE', SURF_ID='INERT'/ 
&OBST ID='AcDb3dSolid - 8A', XB=11.25,13.85,14.45,17.75,9.7,9.8, COLOR='INVISIBLE', SURF_ID='INERT'/ 
&OBST ID='AcDb3dSolid - 8A', XB=13.85,15.95,14.45,17.75,9.7,9.9, COLOR='INVISIBLE', SURF_ID='INERT'/ 
&OBST ID='AcDb3dSolid - 8A', XB=15.95,16.85,14.45,17.75,9.8,9.9, COLOR='INVISIBLE', SURF_ID='INERT'/ 

****Skipped past few 1000 lines of obstructions ********

&HOLE ID='Fire Hole', XB=1.15,2.45,15.35,16.65,1.0,1.8/ 

&VENT ID='Mesh Vent: FIRE MESH BOTTOM [XMIN]', SURF_ID='OPEN', XB=0.95,0.95,14.45,17.75,0.4,3.7/ 
&VENT ID='Mesh Vent: FIRE MESH 2 [XMIN]', SURF_ID='OPEN', XB=0.95,0.95,14.45,17.75,3.7,5.2/ 
&VENT ID='Mesh Vent: FIRE MESH 3 [XMIN]', SURF_ID='OPEN', XB=0.95,0.95,14.45,17.75,5.2,6.7/ 
&VENT ID='Mesh Vent: FIRE MESH COMBINED [XMIN]', SURF_ID='OPEN', XB=0.95,0.95,14.45,17.75,6.7,10.9/ 
&VENT ID='Mesh Vent: FIRE MESH COMBINED [ZMAX]', SURF_ID='OPEN', XB=0.95,16.85,14.45,17.75,10.9,10.9/ 
&VENT ID='Mesh Vent: OUTER MESH 1 [XMAX]', SURF_ID='OPEN', XB=27.95,27.95,0.05,31.55,0.4,10.9/ 
&VENT ID='Mesh Vent: OUTER MESH 1 [YMAX]', SURF_ID='OPEN', XB=16.85,27.95,31.55,31.55,0.4,10.9/ 
&VENT ID='Mesh Vent: OUTER MESH 1 [YMIN]', SURF_ID='OPEN', XB=16.85,27.95,0.05,0.05,0.4,10.9/ 
&VENT ID='Mesh Vent: OUTER MESH 1 [ZMAX]', SURF_ID='OPEN', XB=16.85,27.95,0.05,31.55,10.9,10.9/ 
&VENT ID='Mesh Vent: OUTER MESH 2 [XMIN]', SURF_ID='OPEN', XB=0.95,0.95,0.05,14.45,0.4,10.9/ 
&VENT ID='Mesh Vent: OUTER MESH 2 [YMIN]', SURF_ID='OPEN', XB=0.95,16.85,0.05,0.05,0.4,10.9/ 
&VENT ID='Mesh Vent: OUTER MESH 2 [ZMAX]', SURF_ID='OPEN', XB=0.95,16.85,0.05,14.45,10.9,10.9/ 
&VENT ID='Mesh Vent: OUTER MESH 3 [XMIN]', SURF_ID='OPEN', XB=0.95,0.95,17.75,31.55,0.4,10.9/ 
&VENT ID='Mesh Vent: OUTER MESH 3 [YMAX]', SURF_ID='OPEN', XB=0.95,16.85,31.55,31.55,0.4,10.9/ 
&VENT ID='Mesh Vent: OUTER MESH 3 [ZMAX]', SURF_ID='OPEN', XB=0.95,16.85,17.75,31.55,10.9,10.9/ 

&SLCF QUANTITY='VELOCITY', VECTOR=.TRUE., ID='X Vel 1', PBX=14.2/
&SLCF QUANTITY='VELOCITY', VECTOR=.TRUE., ID='X Vel 2', PBX=23.3/
&SLCF QUANTITY='VELOCITY', VECTOR=.TRUE., ID='Y Vel 1', PBY=28.9/
&SLCF QUANTITY='VELOCITY', VECTOR=.TRUE., ID='Y Vel 2', PBY=26.35/
&SLCF QUANTITY='VISIBILITY', ID='X Vis 1', PBX=18.209142/
&SLCF QUANTITY='VISIBILITY', VECTOR=.TRUE., ID='X Vis 2', PBX=13.5/
&SLCF QUANTITY='VISIBILITY', VECTOR=.TRUE., ID='X Vis 3', PBX=14.35/
&SLCF QUANTITY='VISIBILITY', VECTOR=.TRUE., ID='X Vis 4', PBX=23.4/
&SLCF QUANTITY='VISIBILITY', ID='X Vis 5', PBX=9.809142/
&SLCF QUANTITY='VISIBILITY', VECTOR=.TRUE., ID='Y Vis 1', PBY=26.35/
&SLCF QUANTITY='VISIBILITY', VECTOR=.TRUE., ID='Y Vis 2', PBY=29.05/
&SLCF QUANTITY='VISIBILITY', ID='Z Vis 1', PBZ=3.0/
&SLCF QUANTITY='VISIBILITY', ID='Z Vis 2', PBZ=5.8/
&SLCF QUANTITY='VOLUME FRACTION', SPEC_ID='CARBON MONOXIDE', ID='Z CO 1', PBZ=3.0/
&SLCF QUANTITY='VOLUME FRACTION', SPEC_ID='CARBON MONOXIDE', ID='Z CO 2', PBZ=5.8/
&SLCF QUANTITY='TEMPERATURE', ID='Z Temp 1', PBZ=3.0/
&SLCF QUANTITY='TEMPERATURE', ID='Z Temp 2', PBZ=5.8/
&SLCF QUANTITY='PRESSURE', ID='X Press 1', PBX=3.9/
&SLCF QUANTITY='PRESSURE', ID='X Press 2', PBX=9.9/
&SLCF QUANTITY='PRESSURE', ID='X Press 3', PBX=15.9/
&SLCF QUANTITY='PRESSURE', ID='X Press 4', PBX=21.9/
&SLCF QUANTITY='PRESSURE', ID='Y Press 1', PBY=21.9/
&SLCF QUANTITY='PRESSURE', ID='Y Press 2', PBY=3.9/
&SLCF QUANTITY='PRESSURE', ID='Y Press 3', PBY=9.9/
&SLCF QUANTITY='PRESSURE', ID='Y Press 4', PBY=15.9/
&SLCF QUANTITY='PRESSURE', ID='Y Press 5', PBY=27.9/
&SLCF QUANTITY='PRESSURE', ID='Z Press 1', PBZ=3.0/
&SLCF QUANTITY='PRESSURE', ID='Z Press 2', PBZ=5.8/
&SLCF QUANTITY='PRESSURE', ID='Z Press 3', PBZ=7.3/
&SLCF QUANTITY='PRESSURE', ID='Z Press 4', PBZ=9.4/
&SLCF QUANTITY='PRESSURE', ID='Z Press 5', PBZ=8.2/
&SLCF QUANTITY='PRESSURE', ID='Z Press 6', PBZ=1.9/

&DEVC ID='Pressure_VOLUME MEAN', QUANTITY='PRESSURE', SPATIAL_STATISTIC='VOLUME MEAN', XB=13.55,15.05,16.25,21.95,0.0,10.0/
&DEVC ID='Pressure_MIN', QUANTITY='PRESSURE', SPATIAL_STATISTIC='MIN', XB=13.55,15.05,16.25,21.95,0.0,10.0/
&DEVC ID='Pressure_MAX', QUANTITY='PRESSURE', SPATIAL_STATISTIC='MAX', XB=13.55,15.05,16.25,21.95,0.0,10.0/
&DEVC ID='Pressure_MEAN', QUANTITY='PRESSURE', SPATIAL_STATISTIC='MEAN', XB=13.55,15.05,16.25,21.95,0.0,10.0/
&DEVC ID='Pressure01_VOLUME MEAN', QUANTITY='PRESSURE', SPATIAL_STATISTIC='VOLUME MEAN', XB=11.75,18.95,28.25,29.75,-7.21645E-16,10.0/
&DEVC ID='Pressure01_MIN', QUANTITY='PRESSURE', SPATIAL_STATISTIC='MIN', XB=11.75,18.95,28.25,29.75,-7.21645E-16,10.0/
&DEVC ID='Pressure01_MAX', QUANTITY='PRESSURE', SPATIAL_STATISTIC='MAX', XB=11.75,18.95,28.25,29.75,-7.21645E-16,10.0/
&DEVC ID='Pressure01_MEAN', QUANTITY='PRESSURE', SPATIAL_STATISTIC='MEAN', XB=11.75,18.95,28.25,29.75,-7.21645E-16,10.0/
&DEVC ID='Pressure02_VOLUME MEAN', QUANTITY='PRESSURE', SPATIAL_STATISTIC='VOLUME MEAN', XB=20.75,24.05,21.35,27.05,-1.44329E-15,10.0/
&DEVC ID='Pressure02_MIN', QUANTITY='PRESSURE', SPATIAL_STATISTIC='MIN', XB=20.75,24.05,21.35,27.05,-1.44329E-15,10.0/
&DEVC ID='Pressure02_MAX', QUANTITY='PRESSURE', SPATIAL_STATISTIC='MAX', XB=20.75,24.05,21.35,27.05,-1.44329E-15,10.0/
&DEVC ID='Pressure02_MEAN', QUANTITY='PRESSURE', SPATIAL_STATISTIC='MEAN', XB=20.75,24.05,21.35,27.05,-1.44329E-15,10.0/


&TAIL /

**Comment by drjfloyd** [comment]
If you are running this through Pyrosim then 2023.3.1312 is running FDS 6.8.0 and not the current release of 6.9.1. If you aren't running the current release, you should try it as we may have fixed something between 6.8.0 and 6.9.1.

**Comment by mcgratta** [comment]
This case runs using the latest source code. Try using the latest official release, but if you still have trouble, try the latest nightly build.
